src/templates/games/game_form.html

```diff
@@ -4,10 +4,72 @@
   Games
 {% endblock title %}
 {% block content %}
-  <h1>Update or Create Game</h1>
-  <form method="post">
+  <h1>
+    {% if is_create %}
+      Create
+    {% else %}
+      Update
+    {% endif %}
+    Game
+  </h1>
+  <!-- Game Form -->
+  <form method="post" id="game_form">
     {% csrf_token %}
-    {{ form.as_p }}
+    <!-- Manually render each form field -->
```

Comment:
Adding a statement that justifies manually creating this form would be useful.
I personally imagine its to add the "search BGG" button.